modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc
+//* installing-vsphere-installer-provisioned-network-customizations.adoc
+//* installing_vsphere/installing-vsphere.adoc
+//* installing-vsphere-network-customizations.adoc
+//* installing-restricted-networks-installer-provisioned-vsphere
+//* installing-restricted-networks-vsphere.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere regions and zones
+
+You can install an {product-title} cluster in a vSphere environment that consists of multiple datacenters running in a single VMware vCenter. 
+
+By specifying multiple datacenters to run in a single VMware vCenter, you can reduce the risk of a hardware failure or network outage causing your cluster to fail.
+
+You can also specify multiple regions and zones on a cluster after installation. 
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+* Region: Specifies a vCenter datacenter. You define a region by using a tag from a created tag category, such as `openshift-region`. 
+* Zone: Specifies a vCenter cluster. You define a region by using a tag from from a created tag category, such as `openshift-zone`. 
+
+[NOTE]
+====
+If you plan on specifying more than one failure domain in your `install-config.yaml` file, you must create tag categories, zone tags, and region tags in advance of creating the configuration file.
+====
+
+For {product-title} {product-version}, the default `install-config.yaml` file includes `vcenters` and `failureDomains` fields, so that you can choose to specify multiple datacenters, region, and zone information for your cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter running in a VMware vCenter.
+
+[NOTE]
```

Comment:
I plan on proposing the the content around defining regions and zones be broken into a separate procedure module., In light of that comment, I think it might be OK to remove this note entirely.